packaging/suse/datadog-agent.init

```diff
@@ -417,7 +418,12 @@ case "$1" in
         DOGSTATSD_RETURN=$?
         su $AGENTUSER -c "$FORWARDERPATH info"
         FORWARDER_RETURN=$?
-        exit $(($COLLECTOR_RETURN+$DOGSTATSD_RETURN+$FORWARDER_RETURN))
+        TRACEAGENT_RETURN=0
+        if [ -x $TRACEAGENTPATH ];
+           su $AGENTUSER -c "$TRACEAGENTPATH -info"
+           TRACEAGENT_RETURN=$?
+        fi
+        exit $(($COLLECTOR_RETURN+$DOGSTATSD_RETURN+$FORWARDER_RETURN+$TRACEAGENT_RETURN))
```

Comment:
Yes @truthbk really makes sense -> @talwai do we have a way to do that beyond parsing the conf for `apm_enable=true` & reading the env var `DD_APM_ENABLED` ?